Runtime/Scripts/DracoNative.cs

```diff
@@ -804,7 +811,39 @@ class AttributeMap : AttributeMapBase {
             }
         }
 
-        
+        /// <summary>
+        /// Attribute map for the color attribute, with an increased numcomponents for when the attribute is RGB and byte or ushort
+        /// Unity specifies attribute data size must be multiple of 4
+        /// </summary>
+        class PaddedColorAttributeMap : AttributeMap
```

Comment:
I initially went with color _only_ since it would produce "expected" results; I'm not sure what other conditions would produce this issue in Unity. For example, if we receive texture coordinates as byte (X,Y) I'm not sure how Unity will interpret this when padded as XYZW to meet the 4 byte limitation?  

Thinking about it further the above scenario is probably fine, so I'll adjust this too.

Reply:
I have access to Unity's source code, so I can confirm it is a generic limitation.
Thanks!